### PR TITLE
rename "master" to "main"

### PR DESCRIPTION
**Pull request by jvanasco**
This PR replaces all the relevant "master" branch usage to "main" in Docs and CI.

See:: https://github.com/Pylons/pyramid/issues/3712

### CODE REVIEW

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Please check the failing builds and resolve.

**Comment by jvanasco** [comment]
I think the Windows build failure on the first commit was just an ephemeral network issue.

The second commit fixed the docs failure by replacing an outdated cross-reference to the SQLAlchemy docs with an appropriate candidate.

File: docs/tutorials/wiki2/installation.rst [hunk 1]
```diff
@@ -273,7 +273,7 @@ On Windows
 
 There should be no output to your console.
 You should now have a ``tutorial.sqlite`` file in your current working directory.
-This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, and ``master``, where the first two tables each have single record inside of them.
+This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, and ``main``, where the first two tables each have single record inside of them.
```
Comment (mmerickel) [inline docs/tutorials/wiki2/installation.rst:276]: I ran the cookiecutter and do not see a master/main table created, not sure where that language came from in the tutorial.

Comment (stevepiercy) [reply docs/tutorials/wiki2/installation.rst:276]: Yeah, I don't know where that came from, but it was at least 4 years ago. This should read:

```
This is an SQLite database with two tables defined in it, ``alembic_version`` and ``models``, where each table has a single record.

Comment (mmerickel) [reply docs/tutorials/wiki2/installation.rst:276]:
```suggestion
This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, where the first two tables each have single record inside of them.
```

Comment (stevepiercy) [reply docs/tutorials/wiki2/installation.rst]: That should be this, as I mentioned earlier:

```
This is an SQLite database with two tables defined in it, ``alembic_version`` and ``models``, where each table has a single record.
```

Comment (mmerickel) [reply docs/tutorials/wiki2/installation.rst]: Oops thanks for following up on it.

**Pull request merged**